.env.example

```diff
@@ -1,4 +1,4 @@
 DATABASE_URL=postgres://postgres:canduma@localhost/canduma
-SECRET=GNLw1GKzykA926Rhdcpy1c6lugZXTd5y
-DOMAIN=localhost
+AUTH_SECRET_KEY=GNLw1GKzykA926Rhdcpy1c6lugZXTd5y
+DOMAIN=canduma.localhost.tv
```

Comment:
My test workflow tool was bugging me about the fact than `localhost` was not a suitable host for Cookies...

Reply:
Thank you, I will specify it in the readme !